modules/ROOT/nav.adoc

```diff
@@ -14,6 +14,7 @@
 
 * Manage data
 ** xref::nos.adoc[]
+** xref:ELT:getting-started-with-airbyte.adoc[]
```

Comment:
The name of the adoc file becomes the URL. We need to think about SEO here. Perhaps `use-airbyte-to-load-data-from-external-sources-to-teradata vantage.adoc` will work better.

Reply:
Yes, I have made the update.